### PR TITLE
removed old/uneeded code

### PR DESCRIPTION
**Pull request by yajinni**

### CODE REVIEW

**Comment by yajinni** [comment]
Removed a few lines from the  crimson scorge code
Updated change log
Updates patch compatibility. Josh and me agree its 8.0.1 ready

File: src/Parser/DeathKnight/Blood/CHANGELOG.js [hunk 1]
```diff
@@ -7,6 +7,16 @@ import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 
 export default [
+  {
+    date: new Date('2018-07-19'),
+    changes: <React.Fragment>Updated <SpellLink id={SPELLS.OSSUARY_TALENT.id} /> to its new reduced value.</React.Fragment>,
```
Comment (Gebuz) [inline src/Parser/DeathKnight/Blood/CHANGELOG.js]: Is this meant to be Foul Bulwark instead of Ossuary? afaik Ossuary didn't change.

Comment (yajinni) [reply src/Parser/DeathKnight/Blood/CHANGELOG.js]: Yes wierd, idk how i messed that up... fixed

**Comment by yajinni** [comment]
fixed

**Comment by yajinni** [comment]
Another pull request was accepted that had most of this code so closing this.

**Pull request closed without merging**